cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/ScanApplicationsRequest.java

```diff
@@ -50,12 +53,14 @@ public class ScanApplicationsRequest {
    * @param limit        maximum number of records to return
    */
   private ScanApplicationsRequest(@Nullable NamespaceId namespaceId,
+                                  @Nullable ApplicationReference applicationReference,
```

Comment:
Please change to `String application` and add a javadoc

Reply:
Currently it's partially duplicating namespaceId param

Reply:
updated